python/dgl/dataloading/base.py

```diff
@@ -475,6 +482,11 @@ def as_edge_prediction_sampler(
 
         * User-defined exclusion rule. It is a callable with edges in the current
           minibatch as a single argument and should return the edges to be excluded.
+    exclude_eids : Tensor or dict[etype, Tensor], optional
```

Comment:
The docstring is a bit confusing. In the docstring of the `exclude` argument, it says setting it to `None` means excluding no edge, but this is not true anymore when the user provides `exclude_eids`.